app/models/tagset_models/tag_nomination.rb

```diff
@@ -68,19 +68,19 @@ def set_tag_status
 
   before_save :set_parented, unless: :blank_tagname?
   def set_parented
-    if type == "FreeformNomination"
-      # skip freeforms
-      self.parented = true
-    elsif (tag = Tag.find_by_name(tagname)) &&
-      ((!tag.parents.empty? && get_parent_tagname.blank?) || tag.parents.collect(&:name).include?(get_parent_tagname))
-      # if this is an existing tag and has matching parents, or no parent specified and it already has one
-      self.parented = true
-      self.parent_tagname ||= get_parent_tagname
-    else
+    self.parented = true and return if type == "FreeformNomination"
+
+    self.parent_tagname ||= get_parent_tagname
+
+    tag = Tag.find_by(name: tagname)
```

Comment:
`find_by(name: )` and `find_by_name()` behave differently for tags. Is this a change we absolutely need here? While it's probably okay to make this change because we're not working with URL parameters, there's something to be said for consistency: we only use `Tag.find_by(name:)` in about 3 places and `Tag.find_by_name()` in about 16.